docs/source/events/payment-status-change.rst

```diff
@@ -5,5 +5,26 @@ This event means that the status of a payment has changed. To reduce noise, we o
 
 For each payment method, there are typical status sequences that the payment goes through. The exact meaning of the statuses and content of the ``payment_data`` also depends on the payment method.
 
+
+Payment statuses
+----------------
+
+Here is a list of the most common payment statuses used in Impact Stack 1.0.
+
+- ``payment_status_success``: The payment was successfully processed. The donation form was submitted and the supporter was redirected to the thank you page.
+- ``payment_status_failed``: Something went wrong during the payment.
+- ``payment_status_new``: The payment process was (re)started reusing the same form submission.
+- ``payment_status_pending``: The payment is being processed by the payment provider.
+
+For Stripe there are a few additional statuses:
+
+- ``stripe_payment_status_accepted``: Our backend was notified that Stripe validated and approved the payment. The form submission is continued which triggers the ``payment_status_success`` status. These two statuses usually occur almost at the same time and might be emitted in any order.
```

Comment:
We have a note in our Notion notes to say that this doesn't necessarily mean the bank has accepted/will accept the payment

Reply:
It’s not something I’m sure enough of at the moment to include it here.